### PR TITLE
Update versions in lib/go/test

### PR DESCRIPTION
**Pull request by HouseOfHufflepuff**
Update cadence, flow-sdk, flow-emulator versions in lib/go/test package per:
https://github.com/dapperlabs/nba-smart-contracts/issues/134

### CODE REVIEW

File: lib/go/test/topshot_test.go [hunk 1]
```diff
@@ -398,7 +398,7 @@ func TestMintNFTs(t *testing.T) {
 		assert.Equal(t, cadence.NewBool(true), result)
 
 		result = executeScriptAndCheck(t, b, templates.GenerateGetCollectionIDsScript(env), [][]byte{jsoncdc.MustEncode(cadence.Address(topshotAddr))})
-		idsArray := cadence.NewArray([]cadence.Value{cadence.NewUInt64(1), cadence.NewUInt64(2), cadence.NewUInt64(3), cadence.NewUInt64(4), cadence.NewUInt64(5), cadence.NewUInt64(6)})
+		idsArray := cadence.NewArray([]cadence.Value{cadence.NewUInt64(6), cadence.NewUInt64(4), cadence.NewUInt64(2), cadence.NewUInt64(1), cadence.NewUInt64(5), cadence.NewUInt64(3)})
```
Comment (HouseOfHufflepuff) [inline lib/go/test/topshot_test.go:401]: @joshuahannan result is now returning as [6, 4, 2, 1, 5, 3] each run. Explored assert.Contains but that was not playing well with cadence.value. Any suggestions appreciated, or perhaps this is the only way to do it.

Comment (joshuahannan) [reply lib/go/test/topshot_test.go:401]: I think the latest version of Cadence had a change to how arrays are ordered, or something like that... I think this is fine.

**Review by joshuahannan — APPROVED** [review]
Thanks for doing this! There actually was another emulator release yesterday: Flow Emulator v0.27.0
Could you update to that one instead? The cadence and flow-go-sdk versions will stay the same.

**Comment by joshuahannan** [comment]
Thanks! Merging now

**Pull request merged**